From d7fc3c3dc11413d87b36bebb884938427f4b933a Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Thu, 4 Apr 2019 16:46:36 +0800
Subject: [PATCH 08/30] drm/imx/dcss: use new name
 HDMI_EXTENDED_COLORIMETRY_OPRGB

In upstream commit 391d6f13307e ("media: hdmi.h: rename ADOBE_RGB
to OPRGB and ADOBE_YCC to OPYCC"), HDMI_EXTENDED_COLORIMETRY_ADOBE_RGB
has been renamed to HDMI_EXTENDED_COLORIMETRY_OPRGB.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/gpu/drm/imx/dcss/dcss-crtc.c |  2 +-
 drivers/gpu/drm/imx/hdp/imx-hdmi.c   | 12 ++++++------
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/drivers/gpu/drm/imx/dcss/dcss-crtc.c b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
index 47303b4ccebb..b310ea966947 100644
--- a/drivers/gpu/drm/imx/dcss/dcss-crtc.c
+++ b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
@@ -191,7 +191,7 @@ void dcss_crtc_setup_opipe(struct drm_crtc *crtc, struct drm_connector *conn,
 	if ((colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_BT2020)) ||
 	    (colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_BT2020_CONST_LUM)))
 		dcss_crtc->opipe_g = G_REC2020;
-	else if ((colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_ADOBE_RGB)) &&
+	else if ((colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_OPRGB)) &&
 		 !(crtc->state->adjusted_mode.private_flags & YUV_MODE))
 		dcss_crtc->opipe_g = G_ADOBE_ARGB;
 	else
diff --git a/drivers/gpu/drm/imx/hdp/imx-hdmi.c b/drivers/gpu/drm/imx/hdp/imx-hdmi.c
index 952df47f3385..a3d53bf424da 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdmi.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdmi.c
@@ -24,10 +24,10 @@
 static int character_freq_khz;
 
 #define RGB_ALLOWED_COLORIMETRY (BIT(HDMI_EXTENDED_COLORIMETRY_BT2020) |\
-				 BIT(HDMI_EXTENDED_COLORIMETRY_ADOBE_RGB))
+				 BIT(HDMI_EXTENDED_COLORIMETRY_OPRGB))
 #define YCC_ALLOWED_COLORIMETRY (BIT(HDMI_EXTENDED_COLORIMETRY_BT2020) |\
 				 BIT(HDMI_EXTENDED_COLORIMETRY_BT2020_CONST_LUM) |\
-				 BIT(HDMI_EXTENDED_COLORIMETRY_ADOBE_YCC_601) |\
+				 BIT(HDMI_EXTENDED_COLORIMETRY_OPYCC_601) |\
 				 BIT(HDMI_EXTENDED_COLORIMETRY_S_YCC_601) |\
 				 BIT(HDMI_EXTENDED_COLORIMETRY_XV_YCC_709) |\
 				 BIT(HDMI_EXTENDED_COLORIMETRY_XV_YCC_601))
@@ -57,12 +57,12 @@ static int hdmi_avi_info_set(struct imx_hdp *hdp,
 		ext_colorimetry = HDMI_EXTENDED_COLORIMETRY_BT2020;
 	else if (sink_colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_BT2020_CONST_LUM))
 		ext_colorimetry = HDMI_EXTENDED_COLORIMETRY_BT2020_CONST_LUM;
-	else if (sink_colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_ADOBE_RGB))
-		ext_colorimetry = HDMI_EXTENDED_COLORIMETRY_ADOBE_RGB;
+	else if (sink_colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_OPRGB))
+		ext_colorimetry = HDMI_EXTENDED_COLORIMETRY_OPRGB;
 	else if (sink_colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_XV_YCC_709))
 		ext_colorimetry = HDMI_EXTENDED_COLORIMETRY_XV_YCC_709;
-	else if (sink_colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_ADOBE_YCC_601))
-		ext_colorimetry = HDMI_EXTENDED_COLORIMETRY_ADOBE_YCC_601;
+	else if (sink_colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_OPYCC_601))
+		ext_colorimetry = HDMI_EXTENDED_COLORIMETRY_OPYCC_601;
 	else if (sink_colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_S_YCC_601))
 		ext_colorimetry = HDMI_EXTENDED_COLORIMETRY_S_YCC_601;
 	else if (sink_colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_XV_YCC_601))
-- 
2.17.1

